data/net/minecraft/client/renderer/ScreenEffectRenderer.mapping

```diff
@@ -2,11 +2,14 @@ CLASS net/minecraft/client/renderer/ScreenEffectRenderer
 	METHOD getViewBlockingState (Lnet/minecraft/world/entity/player/Player;)Lnet/minecraft/world/level/block/state/BlockState;
 		ARG 0 player
 	METHOD renderFire (Lnet/minecraft/client/Minecraft;Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 0 minecraft
-		ARG 1 matrixStack
+		ARG 0 mc
+		ARG 1 poseStack
 	METHOD renderScreenEffect (Lnet/minecraft/client/Minecraft;Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 0 minecraft
-		ARG 1 matrixStack
+		ARG 0 mc
+		ARG 1 poseStack
+	METHOD renderTex (Lnet/minecraft/client/renderer/texture/TextureAtlasSprite;Lcom/mojang/blaze3d/vertex/PoseStack;)V
+		ARG 0 texture
+		ARG 1 poseStack
 	METHOD renderWater (Lnet/minecraft/client/Minecraft;Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 0 minecraft
-		ARG 1 matrixStack
+		ARG 0 mc
```

Comment:
`mc` was more of a MCP convention, should be changed to `minecraft` to avoid abbreviations and be more consistent with official mappings.

Reply:
Good to know! Fixed